docs/dxp/7.x/en/site-building/creating-pages/master-pages.md

```diff
@@ -0,0 +1,36 @@
+# Master Page Templates
+
+```note::
+  Master Page Templates are available since Liferay DXP 7.3.
+```
+
+A Master page Template provides the common elements for a page (Headers, Footers, etc.), while the Theme provides the overall look and feel for the Site's pages. Prior to Liferay DXP 7.3, these common elements were provided by the Theme, which means that any changes that needed to be made required an update to the Theme. Providing the common elements through a Theme poses a few limitations:
+
+* Only users with System Administrator permissions can apply changes, since they must deploy the Theme.
+
+* Themes are available for all virtual instances and Sites, so you can't make a Theme available to certain customers, while hiding it from others.
+
+* Common Theme element (Headers, Footers, etc.) customization options are limited to what the Theme developer provides (i.e. color schemes, theme settings, etc.).
+
+Master Page Templates solve all these limitations by letting you create common elements that are unique per Site. The remainder of the page is left editable and is built like any other [Content Page](./04-building-content-pages.md). Each Page and Page Template must have an associated Master Page Template. By default, this is the "Blank" Master, which doesn't provide any common elements.
+
+![The Blank Master Page Template is the default for Pages, and Page Templates.](./master-pages/images/01.png)
+
+```note::
+    In Liferay DXP 7.3+, Page Templates and Display Page Templates have been moved out of "Pages" and into their own separate application called "Page Templates" under "Site Builder".
+```
+
+## Master Page Elements
+
+Master Pages are divided into three main sections by default: 
+
+**Header elements:** Where common Header elements are placed.
+
+**Drop Zone:** Defines where you can place Page Fragments. This area can't be removed. 
```

Comment:
I am not sure we should divide the master page into three sections because it doesn't have to be header nor footer. I think we should mention somewhere that the dropzone can be dragged and moved to any section or row. 

For example, you may create a master page, add a row with 2 columns, move the drop zone to one column and add some fragments in the other column, this way you can create a page where you can only add fragments to a column and so on.

I would say something like the typical case for a master page is to have its header, footer, and content in the middle, but don't mention that is divided 🤔 

What do you think? is this sound reasonable?

Reply:
Oh, ok. I didn't know you could drag the dropzone. I'll update it as you suggested. Thanks @victorg1991 !